test/authenticator.spec.ts

```diff
@@ -184,18 +168,16 @@ describe('Decentraland Crypto', function () {
       const result = await Authenticator.validateSignature(
         'QmNUd7Cyoo9CREGsACkvBrQSb3KjhWX379FVsdjTCGsTAz',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/2c902c2e3b8947d3b34bba7ca48635fc'
-        ),
+        new HTTPProvider('https://rpc.decentraland.org/mainnet'),
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?